tests/test_opds_import.py

```diff
@@ -616,16 +645,16 @@ def test_resources_are_mirrored_on_import(self):
 
         http = DummyHTTPClient()
         http.queue_response(
-            200, content='I am 10557.epub.images',
-            media_type=Representation.EPUB_MEDIA_TYPE,
-        )
-        http.queue_response(
             200, content='I am 10441.epub.images',
             media_type=Representation.EPUB_MEDIA_TYPE
         )
         http.queue_response(
             200, content=svg, media_type=Representation.SVG_MEDIA_TYPE
         )
+        http.queue_response(
+            200, content='I am 10557.epub.images',
+            media_type=Representation.EPUB_MEDIA_TYPE,
+        )
```

Comment:
I don't understand how your branch changed it so 10557 gets mirrored last. It looks like you switched from mirroring all the circulation resources, then mirroring all the other resources, to mirroring one book at a time. I like this change but I can't find the specific bit where you changed it. Can you just point it out?